src/nls/messages_en.json

```diff
@@ -66,6 +66,14 @@
     "dashboard.taskRun.status.succeeded": "Completed",
     "dashboard.taskRun.status.waiting": "Waiting",
     "dashboard.triggerBindings.error": "Error:",
-    "dashboard.triggerTemplate.error": "Error:"
+    "dashboard.triggerTemplate.creationTime": "Creation time",
+    "dashboard.triggerTemplate.default": "Default",
+    "dashboard.triggerTemplate.description": "Namespace",
```

Comment:
I believe this should be Description... also, Name and Namespace can be found in 
"dashboard.tableHeader.name": "Name",
"dashboard.tableHeader.namespace": "Namespace"